code/style.css

```diff
@@ -1,7 +1,352 @@
+  background-repeat: no-repeat;
+}
+
+.header-text {
+  margin-top: 300px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.backdrop {
+  background-color: rgba(255, 255, 255, 0.3);
+  border-radius: 5px;
+  backdrop-filter: blur(10px);
+  -webkit-backdrop-filter: blur(10px);
+}
+
+h1 {
+  font-family: "Alata", sans-serif;
+  color: white;
+  font-size: 60px;
+  text-align: center;
+  margin: 20px;
+}
+
+.header-text p {
+  font-family: "Alata", sans-serif;
+  width: 400px;
+  color: white;
+  background-color: #00917c;
+  text-align: center;
+  font-size: 20px;
+  font-weight: bold;
+  border-radius: 5px;
+  margin-top: 5px;
+}
+
+/* FORM STYLING */
+form {
+  width: 40%;
+  max-width: 600px;
+  min-width: 450px;
+  font-family: "Roboto", sans-serif;
+  font-size: 18px;
+  font-weight: bold;
+  margin: 0 20px;
+}
+
+fieldset {
+  background-color: #bea69f;
+  border: none;
+  border-radius: 7px;
+}
+
+legend {
+  font-family: "Alata", sans-serif;
+  font-size: 25px;
+  font-weight: bold;
+  color: white;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.details {
+  margin-top: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.details label {
+  margin-bottom: 17px;
+}
+
+.details input {
+  font-size: 20px;
+  border-radius: 3px;
+  border: 3px solid white;
+}
+
+.details input:focus {
+  outline: none;
+  border: 3px solid #00917c;
+}
+
+select {
+  font-size: 18px;
+  border: none;
+}
+
+.form p {  /* modifies the text:"number of coffee variations..." */
+  margin-bottom: 5px;
+}
+
+.radio-buttons {
+  display: flex;
+  flex-direction: row;
+}
+
+.radio-button-container {
+  display: block;
+  position: relative;
+  padding-left: 25px;
+  margin: 0 15px 10px 15px;
+  font-size: 16px;
+  user-select: none;
+}
+
+/* hide the radio button with the default styling */
+.radio-button-container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+/* custom radio buttons */
+.custom-radio {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 19px;
+  width: 19px;
+  background-color: #eee;
+  cursor: pointer;
+  border-radius: 50%;
+}
+
+.radio-button-container:hover input ~ .custom-radio {
+  background-color: #ccc;
+}
+
+.radio-button-container input:checked ~ .custom-radio {
+  background-color: #00917c;
+}
+
+.custom-radio:after {
+  /* creates the white dot in the middle, hidden now */
+  content: "";
+  position: absolute;
+  top: 6px;
+  left: 6px;
+  width: 7px;
+  height: 7px;
+  background-color: #fff;
+  border-radius: 50%;
+  display: none;
+}
+
+.radio-button-container input:checked ~ .custom-radio:after {  /* makes white dot in middle visible after clicking it */
+  display: block;
+}
+
+button { /* styles submit button */
+  margin-top: 25px;
+  font-family: "Alata", sans-serif;
+  font-size: 20px;
+  background-color: #00917c;
+  color: white;
+  padding: 10px;
+  border: none;
+  cursor: pointer;
+}
```

Comment:
Remember, if you have ANY other button on your page, this will be the style it receives.  That could have unintended consequences.  I would like to see this as a class.